src/ui/components/results/paginationcomponent.js

```diff
@@ -67,6 +67,17 @@ export default class PaginationComponent extends Component {
         this.setState();
       }
     });
+
+    /**
+     * Configuration for the behavior when there are no vertical results.
+     */
+    this._noResults = config.noResults || this.core.globalStorage.getState(StorageKeys.NO_RESULTS_CONFIG);
```

Comment:
Let's break this line up, it's pretty long.

Reply:
Do we need to default to {} if it's not present in the component's config or global storage?

Reply:
yeah missed this